tools/jsii-compliance/package.json

```diff
@@ -16,7 +16,7 @@
     "tablemark": "^2.0.0"
   },
   "devDependencies": {
-    "@types/node": "^10.17.60",
+    "@types/node": "^12.20.19",
```

Comment:
Slightly curious why @types/node is set to 12.20.0 even though we only require 12.7.0 - maybe worth adding a note why in the PR description.

Reply:
The `@types/node` versions are __NOT__ related to the `node` versions in any way. Only the major version number is relevant (`12.x` is for `node12`, `14.x` is for `node14`, etc...).

The `@types/node` packages are generally speaking maintained as if the API for a given node major release are always the same (i.e: once it hit GA, no new APIs get added, modified or removed).

See: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/54465

Reply:
Ah! I was misled by this stackoverflow post [here](https://stackoverflow.com/questions/42035263/relationship-between-the-version-of-node-js-and-the-version-of-types-node)... but after diving deeper I see you are totally correct. Thanks for clarifying!